examples/rollback/rollback.bal

```diff
@@ -14,11 +15,19 @@ public function main() returns error? {
         {updateIndex: 3, stockMnt: -1000},
         {updateIndex: 4, stockMnt: -2000}
     ];
-    // If an error is returned from the `transfer` function,
-    // the error is returned from the `main` and it exits.
-    check transfer(updates);
 
-    return;
+    io:println(transfer(updates));
```

Comment:
Shall we add a comment for this, that the transaction is implicitly rolled back due to the error?

Reply:
That is explained inside `transfer` function

Reply:
Will also add a comment there as well